src/resolvers/userResolver.js

```diff
@@ -231,6 +231,58 @@ const userResolver = {
       );
     },
 
+    changePassword: async(_, { oldPassword, newPassword }, { req }) => {
+      const { password } = await User.findById(req.session.userId);
+      const isOldPassword = await bcrypt.compare(oldPassword, password);
+
+      if(!isOldPassword) {
+        return {
+          errors: [
+            {
+              field: "password",
+              message: "old password does not match",
+            },
+          ],
+        };
+      }
+
+      if(oldPassword === newPassword) {
```

Comment:
I don't think we need to give them an error if both passwords are the same so we can remove this check.